requirements/strategies.txt

```diff
@@ -1,4 +1,4 @@
-fairscale>=0.4.5
+fairscale>=0.4.5, <=0.4.6
 deepspeed<0.6.0
 horovod>=0.21.2,!=0.24.0  # no need to install with [pytorch] as pytorch is already installed
 hivemind>=1.0.1; sys_platform == 'linux'
```

Comment:
These have no max pin. On purpose?

Reply:
probably would need to come with better reg expression, so let's first evaluate the core of this PR and then we can finish this last one :)

Reply:
fixed :rabbit: